src/software/simulation/standalone_simulator.h

```diff
@@ -39,6 +46,49 @@ class StandaloneSimulator
      */
     void setupInitialSimulationState();
 
+    /**
+     * Starts the simulation. If the simulator is already running, this
+     * function does nothing.
+     */
+    void startSimulation();
+
+    /**
+     * Stops the simulation. If the simulator is already stopped, this
+     * function does nothing.
+     */
+    void stopSimulation();
+
+    /**
+     * Sets the slow motion multiplier for the simulation. Larger values
+     * cause the simulation to run in slow motion. For example, a value
+     * of 2.0 causes the simulation to run 2x slower.
+     *
+     * @pre value is >= 1.0
```

Comment:
Maybe should be saying `multiplier is >= 1.0` to make it more explicit.

Reply:
:+1: done